SECURITY.md

```diff
@@ -0,0 +1,25 @@
+# Security Policy
+
+## Supported Versions
+
+We do not support any old versions, all updates for bugs/features/security
+will be released in next planned version.
+
+| Version      | Supported          |
+| ------------ | ------------------ |
+| LATEST only  | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+List of maintainers: https://checkstyle.org/team.html
+Mail-list for any questions: https://checkstyle.org/mailing-lists.html
+You can find admin emails in mail-list.
+You can find emails of maintainers in commits, you can clone our repo and see it by "git log".
+
+If we are not responding, please keep pining us. As final resort create issue at
+https://github.com/checkstyle/checkstyle/issues with note that you have security issue
+
+If the vulnerability is accepted we create issue (without much details) on Github Issue tracker
```

Comment:
`is accepted we create issue` => `is accepted, we will create an issue`

Reply:
fixed.